upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,16 @@
 == Migration Changes
 
+=== Migrating to 16.0.0
+
+==== Proxy environment variables
+
+{project_name} now respects the standard `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables for outgoing
```

Comment:
respects?  Should it be accepts?

Reply:
I believe not. Keycloak is now respecting standard variables.